traffic_ops/app/db/create_tables.sql

```diff
@@ -37,6 +37,7 @@ CREATE EXTENSION IF NOT EXISTS plpgsql WITH SCHEMA pg_catalog;
 
 SET search_path = public, pg_catalog;
```

Comment:
After fd740370f0, the Portal user has gained some permissions it did not have before:

```diff
@@ -111,8 +108,13 @@
 5      DS-REQUEST:READ A_TIMESTAMP
 5      DS-REQUEST:UPDATE       A_TIMESTAMP
 5      DS-SECURITY-KEY:READ    A_TIMESTAMP
+5      FEDERATION:CREATE       A_TIMESTAMP
+5      FEDERATION:DELETE       A_TIMESTAMP
 5      FEDERATION:READ A_TIMESTAMP
+5      FEDERATION-RESOLVER:CREATE      A_TIMESTAMP
+5      FEDERATION-RESOLVER:DELETE      A_TIMESTAMP
 5      FEDERATION-RESOLVER:READ        A_TIMESTAMP
+5      FEDERATION:UPDATE       A_TIMESTAMP
 5      ISO:READ        A_TIMESTAMP
 5      JOB:CREATE      A_TIMESTAMP
 5      JOB:DELETE      A_TIMESTAMP
```

Reply:
Okay, I think I got it for  real this time.